store/copr/batch_coprocessor.go

```diff
@@ -827,8 +867,22 @@ func buildBatchCopTasksCore(bo *backoff.Backoffer, store *kvStore, rangesForEach
 			}
 		}
 
+		if !tiflashReplicaReadPolicy.IsPolicyAllReplicas() && isTiDBLabelZoneSet {
+			allTiflashStores := cache.RegionCache.GetTiFlashStores(tikv.LabelFilterNoTiFlashWriteNode)
+			aliveStores = filterAliveStores(bo.GetCtx(), allTiflashStores, ttl, store)
```

Comment:
Since `filterAliveStores` is already called here, there is no need to filter alive stores again in `balanceBatchCopTask` when tiflashReplicaReadPolicy is not all replicas?